rules-base/api/src/main/java/org/jboss/windup/rules/files/condition/ToFileModelHandler.java

```diff
@@ -35,22 +35,14 @@
     @Override
     public ToFileModel processElement(ParserContext context, Element element) throws ConfigurationException
     {
-        // Read & Validate
         List<Element> children = JOOX.$(element).children().get();
         validateChildren(children);
         Element firstChild = children.get(0);
-        Object wrappedCondition = context.processElement(firstChild);
-        validateWrappedCondition(wrappedCondition);
+        GraphCondition wrappedCondition = (GraphCondition) context.processElement(firstChild);
 
-        // Create the condition
-        ToFileModel toFileModelCondition = new ToFileModel();
-        for (Element child : children)
-        {
-            Object condition = context.processElement(child);
-            toFileModelCondition.withWrappedCondition((GraphCondition) condition);
```

Comment:
this was a bug found. withWrappedCondition returns a new Condition and does not change the current one (the method is static)